### PR TITLE
Galaxis: init at 1.9

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Motivation for this change

New package

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/galaxis/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  name = "galaxis-${version}";
+  version = "1.9";
+
+  src = fetchurl{
+    url = "http://www.catb.org/~esr/galaxis/${name}.tar.gz";
+    sha256 = "1dsypk5brfbc399pg4fk9myyh5yyln0ljl1aiqkypws8h4nsdphl";
+  };
+
+  buildInputs =
```
Comment (pSub) [inline pkgs/games/galaxis/default.nix]: Nitpick: Why is there a extra newline? IMO it does not improve readability.

**Review by pSub — APPROVED** [review]

**Comment by pSub** [comment]
@Mic92 I'm curious: I suppose you don't have push access to @AndersonTorres fork, so how did you add commits to this PR?

**Comment by Mic92** [comment]
This is a recent feature of github. When you open a pull request, maintainers have write access to the branch by default, except if contributor untick "Allow edits from maintainers".

**Comment by pSub** [comment]
Ah, good to know. Thank you!

**Pull request merged**